File: 67_Add Binary.py
class Solution:
    def addBinary(self, a: str, b: str) -> str:
        x = int(a, 2) #二进制转十进制
        y = int(b, 2)
        return bin(x + y)[2:] #十进制转二进制，但转完是0b10... 所以从第三位开始输出

File: Algorithms_C#/67_Add_Binary.cs
public class Solution {
    public string AddBinary(string a, string b) {
        StringBuilder ans = new StringBuilder();
        int i = a.Length - 1, j = b.Length - 1;
        for (int carry = 0; i >= 0 || j >= 0 || carry > 0; --i, --j){
            carry += (i >= 0 ? a[i] - '0' : 0) + (j >= 0 ? b[j] - '0' : 0);
            ans.Append(carry % 2);
            carry /= 2;
        }
        StringBuilder ans2 = new StringBuilder();
        for (int k = ans.Length - 1; k >= 0; --k){
            ans2.Append(ans[k]);
        }
        return ans2.ToString();
    }
}